app/Http/Controllers/Api/AssetsController.php

```diff
@@ -524,6 +524,10 @@ public function selectlist(Request $request) : array
             $assets = $assets->RTD();
         }
 
+        if ($request->filled('assetStatusType') && $request->input('assetStatusType') === 'Deployed') {
```

Comment:
We already handle this case here I think.

`status` has a switch case for `Deployed` on line 239

Reply:
Not sure to get the point, as index() and selectlist() functions are not calling each other. Or perhaps that's your point? Calling index() from selectlist()?